### PR TITLE
Activity component

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

**Comment by justinvdm** [comment]
@peterp depends on #74, otherwise ready for review.

**Comment by justinvdm** [comment]
screen shot 2016-08-28 at 6 29 47 pm
screen shot 2016-08-28 at 6 32 27 pm
screen shot 2016-08-28 at 6 32 34 pm
screen shot 2016-08-28 at 6 32 44 pm

## when poster isn't given

screen shot 2016-08-28 at 6 34 14 pm

File: src/activities/Activity/index.js [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+const Section = ({
+  color,
+  icon,
+  title,
+  children,
+}) => (
+  <View style={styles.section}>
+    <View style={styles.sectionHeader}>
+      {/* TODO something better here once this has been discussed */}
+      <Image source={icon} style={[styles.sectionIcon, { tintColor: color }]} />
```
Comment (justinvdm) [inline src/activities/Activity/index.js]: This isn't ideal, but the only alternative I can think of involves quite a bit of modification of the icons. We need to support icons of the category's colour (which is dynamic). Probably something we need to discuss.

**Comment by justinvdm** [comment]
Padding for sections got broken at some point, fixed it:

screen shot 2016-08-29 at 6 57 45 pm

**Comment by peterp** [comment]
👍 , once the merge conflict is sorted! 🍰  :wizard:

**Pull request merged**